templates/home/_carousel.html.twig

```diff
@@ -23,7 +23,7 @@
                 </div>
             </div>
             <div class="carousel-item">
-                <img class="caroussel_image d-block w-100 vh-100" src="/images/Bateau.webp" alt="Third slide">
+                <img class="caroussel_image d-block w-100 vh-100" src="/images/bateau.webp" alt="Third slide">
```

Comment:
du coup je ne l'ai pas vu hier, mais il manque le {{assets('path/to/image') }} comme je l'expliquait tout à l'heure. Je vous laisse l'ajouter à ce fix